src/blueapi/cli/cli.py

```diff
@@ -50,14 +59,25 @@ def main(ctx: click.Context, config: Path | None | tuple[Path, ...]) -> None:
             else:
                 raise FileNotFoundError(f"Cannot find file: {path}")
 
+    # Step 3: Load values from environment variables
+    config_loader.use_values_from_env(env_prefix)
+
+    # Step 4: Load CLI arguments as overrides
+    config_loader.use_values_from_cli(ctx.params)
```

Comment:
Does this work? Doesn't click prevent you from passing anything other than `--config` or `--env_prefix`

Reply:
I'd need to look into this

Reply:
just deleted